inspirehep/ext/jinja_filters/record.py

```diff
@@ -390,3 +390,11 @@ def citation_phrase(count):
             return 'Cited 1 time'
         else:
             return 'Cited ' + str(count) + ' times'
+
+    @app.template_filter()
+    def long_timeout(timeout):
+        return 5184000  # 60 days
+
+    @app.template_filter()
+    def in_debug_mode():
```

Comment:
This is unused it seems? Might be enough to just do in templates `if config.DEBUG` without a filter.